espresso-server/app/src/test/java/io/appium/espressoserver/test/helpers/w3c/ProcessorTest.java

```diff
@@ -122,30 +122,6 @@ public void shouldRejectPointerMoveIfNegativeDuration() throws InvalidArgumentEx
         }
     }
 
-    @Test
-    public void shouldRejectPointerMoveIfNegativeX() throws InvalidArgumentException {
-        Action action = new Action();
-        action.setX(-100);
-        try {
-            processPointerMoveAction(action, InputSourceType.POINTER, "any", 0);
```

Comment:
I don't see any changes regarding this verification

Reply:
It was a failing test from previous changed. It just wasn't caught because these tests don't run on CI. The issue is that I had made it fail for negative x y but it shouldn't automatically be doing that because you can have negative x y if it's a relative origin.